django_browserid/tests/test_views.py

```diff
@@ -132,6 +143,23 @@ def test_auth_success_redirect_success(self):
         self.assert_json_equals(response.content,
                                 {'email': 'test@example.com', 'redirect': '/success'})
 
+    @mock_browserid('test@example.com')
+    def test_auth_success_redirect_success_named_url(self):
+        """
+        If authentication succeeds, redirect to the success URL, and resolve
+        named urls.
+        """
+        user = auth.models.User.objects.create_user('asdf', 'test@example.com')
+
+        request = self.factory.post('/browserid/verify', {'assertion': 'asdf'})
+        with self.settings(LOGIN_REDIRECT_URL='epic_fail'):
+            with patch('django_browserid.views.auth.login') as login:
```

Comment:
I don't _think_ patching this is necessary; we only patched it in the view above since we wanted to assert against it.